filters/local.properties

```diff
@@ -91,3 +91,5 @@ environment.build.ldap.url=
 environment.build.ldap.baseDn=
 environment.build.ldap.userName=
 environment.build.ldap.password=
+
+secureSessionCookie=false
```

Comment:
You should probably set a value in each filters/*.properties file, not just local.properties so it is more likely someone sees it and changes it if need be, and they can be set differently (I assume prod would default to true).